include/rcpputils/filesystem_helper.hpp

```diff
@@ -190,6 +197,28 @@ inline bool create_directories(const path & p)
 
 #undef RCPPUTILS_IMPL_OS_DIRSEP
```

Comment:
Could you move your function before this `#undef` with the other ones, to keep them together?

Reply:
and Done